examples/rpt/count_calculation/rpt_count_calculation.prm

```diff
@@ -0,0 +1,37 @@
+# Listing of Parameters
```

Comment:
In general in this part maybe it is better to add the unit for non dimensionless parameters like radius which is in meter for example!

Reply:
Yeah but there is no way to have units in the PRM file. I mean, we cannot enforce it. We will need to document it that the units have to be consistent (in SI).

Reply:
I will in the documentation and/or the example of the wiki